jobserver/management/commands/check_rap_api_status.py

```diff
@@ -1,14 +1,66 @@
+import datetime
```

Comment:
I don't think it really makes a difference here, but a common convention is to do `from datetime import datetime`, so that references in your code go from `datetime.datetime.fromisoformat` to `datetime.fromisoformat`.